### PR TITLE
improving the position of the components prevent drop menu to overrid…

### PR DESCRIPTION
**Pull request by khaledkz**
…e them

### CODE REVIEW

File: client/src/components/About/about.css [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.About{
```
Comment (kabaros) [inline client/src/components/About/about.css]: small letters for class name (also use dashes, so `about-page` as opposed to `aboutPage`)

File: client/src/components/Categories/CategoriesOverview/category.css [hunk 2]
```diff
@@ -91,4 +91,10 @@
     font-size: 1.2em;
     padding: 0 .5em;
     font-weight: bold;
+}
+
+#root .SingleCategory{
```
Comment (kabaros) [inline client/src/components/Categories/CategoriesOverview/category.css]: same here - wrong naming convention for css `single-category`

File: client/src/containers/Setting/Setting.js [hunk 3]
```diff
@@ -12,7 +12,7 @@ class Setting extends Component {
     render() {
         var localLang = localStorage.getItem("language");
         return (
-            <div className="row">
+            <div className="row app-select-language">
```
Comment (kabaros) [inline client/src/containers/Setting/Setting.js]: I don't understand what you changed here - how the does Language Selector look like now?

**Review by kabaros — CHANGES_REQUESTED** [review]

**Review by kabaros — CHANGES_REQUESTED** [review]
send me screenshots please

**Comment by khaledkz** [comment]
image

**Comment by khaledkz** [comment]
image

**Comment by khaledkz** [comment]
image

**Comment by khaledkz** [comment]
image

**Comment by kabaros** [comment]
I still don't think you need all of this to achieve what you want to achieve - let's review it together on Saturday.

**Review by kabaros — APPROVED** [review]

**Pull request merged**